src/expr/src/explain/text.rs

```diff
@@ -434,6 +445,8 @@ impl MirRelationExpr {
             Project { outputs, input } => {
                 FmtNode {
                     fmt_root: |f, ctx| {
+                        // We deliberately don't print humanized indices because
```

Comment:
Note that I deliberately don't humanize `Project` nodes because we often have quite long projection lists. We can also use a different rule (say humanize projection lists up to size 5 or 10).